rosdep/oe-opkg.yaml

```diff
@@ -0,0 +1,335 @@
+asio:
+  oe:
+    opkg:
+      packages: [asio@meta-oe]
+
+assimp:
+  oe:
+    opkg:
+      packages: [assimp@openembedded-core]
```

Comment:
How does the  `@`  syntax resolve/work here? It's not clear how it looks up the resolution and if I wanted to add a new repository what would be required and what is the policy for where things can be stored?

Reply:
It's <recipe>@<meta-layer>, where <recipe> resides in the OpenEmbedded <meta-layer> and can be queried at http://layers.openembedded.org .

We plan to use that information on superflore to generate a list of dependent layers automatically.